build.gradle

```
@@ -12,7 +12,7 @@ repositories {
 }
 
 group = "io.trocco"
-version = "0.1.8"
+version = "0.1.9.dev2"
```

[nits] please re-fix version ;)
```suggestion
version = "0.1.9"
```